lib/node_modules/@stdlib/array/base/cusome-by-right/benchmark/benchmark.assign.length.js

```diff
@@ -0,0 +1,101 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var isArray = require( '@stdlib/assert/is-array' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var isPositiveInteger = require( '@stdlib/assert/is-positive-integer' ).isPrimitive;
+var filled = require( '@stdlib/array/base/filled' );
+var pkg = require( './../package.json' ).name;
+var cusomeByRight = require( './../lib' );
+
+
+// FUNCTIONS //
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - array length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len ) {
+	var x = filled( 1.5, len );
+	var n = discreteUniform( 1, 0, len );
```

Comment:
This is generating an array, rather than a scalar.

Reply:
It is not clear why we need to randomly generate a value in the first place.

Reply:
can I write this instead,
var n = Math.min( 2, n );

Reply:
Now, I removed the `n` variable and passed len as the second parameter to the API